demisto_sdk/core.py

```diff
@@ -27,7 +31,34 @@ def __init__(self, configuration=Configuration()):
             'Scripts': 'script'
         }
 
-        self.config = configuration
+        self.parser = argparse.ArgumentParser(description='Manage your content with the Demisto SDK.',
+                                              formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        self.subparsers = self.parser.add_subparsers(help='stuff', dest='command')
+        self.initialize_parsers()
+        self.configuration = configuration
+
+    def initialize_parsers(self):
+        Unifier.add_sub_parser(self.subparsers)
+        Extractor.add_sub_parser(self.subparsers)
+        FilesValidator.add_sub_parser(self.subparsers)
+
+    def parse_args(self):
+        args = self.parser.parse_args()
+        if args.command == 'extract':
+            if args.migrate:
+                self.migrate_file(args.infile, args.outfile, args.demistomock, args.commonserver, args.type)
+            else:
+                self.extract_code(args.infile, args.outfile, args.demistomock, args.commonserver, args.type)
+        elif args.command == 'unify':
+            self.unify_package(args.indir, args.outdir)
+        elif args.command == 'validate':
+            if self.validate(is_backward_check=args.backward_comp, is_circle=args.circle,
+                             prev_ver=args.prev_ver, validate_conf_json=args.conf_json, use_git=args.use_git):
+                print_color('The files are valid', LOG_COLORS.GREEN)
+            else:
+                print_color('The files are invalid', LOG_COLORS.RED)
+        else:
+            print('Use demisto_sdk -h for help with the commands.')
```

Comment:
```suggestion
            print('Unknown command, run `demisto_sdk -h` to see the available commands.')
```

Reply:
this message is displayed also when someone types just `demisto_sdk`